src/main/java/heronarts/lx/effect/BlurEffect.java

```diff
@@ -89,6 +89,10 @@ protected void onEnable() {
 
   @Override
   public void run(double deltaMs, double amount) {
+    if (!isEnabled()) {
+      return;
+    }
+
```

Comment:
This shouldn't be there. Effects automatically damp themselves when off and `run()` is not called when that's finished. But it's possible for `isEnabled()` to be false while `amount` is still positive as the effect fades out.

Reply:
Makes sense, thanks.  Changed back.